pkg/cmd/open-svc.go

```diff
@@ -227,15 +231,17 @@ func (o *OpenServiceOptions) Run() error {
 
 	fmt.Printf("Opening service/%s in the default browser...\n", serviceName)
 	for _, url := range urls {
-		if err := browser.OpenURL(url); err != nil {
-			return fmt.Errorf("Failed to open %s in the default browser\n", url)
+		if o.url {
+			fmt.Println(url)
+		} else {
+			if err := browser.OpenURL(url); err != nil {
+				return fmt.Errorf("Failed to open %s in the default browser\n", url)
+			}
 		}
 	}
-
```

Comment:
Please don't make changes which are not relatived this PR.

Reply:
Sorry to trouble you. Probably, Before commit this file, I do `goimoports` to fix  code format. then it lacked I think...